web/src/components/vis/CorrelationTile.vue

```diff
@@ -73,6 +77,32 @@ export default {
             value: Math.abs(d.value),
           }));
     },
+    searchResults() {
+      if (!this.currentUserInput || this.currentUserInput.length === 0) {
+        return [];
+      }
+      return this.searchBarResults;
+    },
+  },
+  mounted() {
+    this.nodeCount = this.nodes.length;
+  },
+  methods: {
+    searchFilter(item, queryText, itemText) {
+      if (itemText.toLocaleLowerCase().indexOf(queryText.toLocaleLowerCase()) > -1) {
+        if (!this.searchBarResults.includes(itemText)) {
+          this.searchBarResults.push(itemText);
+        }
+        return true;
+      }
+      if (this.searchBarResults.includes(itemText)) {
+        this.searchBarResults.splice(this.searchBarResults.indexOf(itemText), 1, '');
+        if (this.searchBarResults.length > this.nodeCount) {
+          this.searchBarResults = this.searchBarResults.filter(result => result !== '');
+        }
+      }
+      return false;
```

Comment:
```suggestion
      const match = itemText.toLocaleLowerCase().indexOf(queryText.toLocaleLowerCase()) > -1;
      if (match) this.searchBarResultsSet.add(itemText);
      else this.searchBarResultsSet.remove(itemText);
      this.searchBarResults = Array.from(this.searchBarResultsSet)
      return match;
```

Naturally my formatting is terrible, but that's the idea.